src/main/kotlin/me/senseiwells/arucas/utils/Trace.kt

```diff
@@ -10,7 +13,7 @@ abstract class Trace(val fileName: String) {
         return "File: ${this.fileName}"
     }
 
-    open fun toString(fileContent: String?): String {
+    open fun toString(interpreter: Interpreter?, message: String?): String {
```

Comment:
From what I can see this interpreter is never null?
But anyways as we discussed I think it's probably better to just store the file contents in each Trace in which case we can remove this interpreter parameter anyways.